ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -58,6 +59,20 @@ @implementation RCTPSPDFKitViewManager
   }
 }
 
+RCT_CUSTOM_VIEW_PROPERTY(leftBarButtonItems, pdfController.navigationItem.leftBarButtonItems, RCTPSPDFKitView) {
```

Comment:
The second parameter expects a type, and not a property. (There are more places where we do this in this file as well.) 

Not 100% sure if that's the correct syntax for an array, but I guess it is:
```suggestion
RCT_CUSTOM_VIEW_PROPERTY(leftBarButtonItems, NSArray<UIBarButtonItem *>, RCTPSPDFKitView) {
```

Reply:
You are right! Fixed them all in https://github.com/PSPDFKit/react-native/pull/233/commits/75c34109e4813a6ba50d2d11cef4a5b246601d81